model_compression_toolkit/exporter/README.md

```diff
@@ -2,31 +2,13 @@
 
 Export your quantized model in the following serialization formats:
 
-* TensorFlow models can be exported as Tensorflow models (`.h5` extension) and TFLite models (`.tflite` extension).
+* TensorFlow models can be exported as Tensorflow models (`.h5` or `.keras` extensions) and TFLite models (`.tflite` extension).
 * PyTorch models can be exported as torch script models and ONNX models (`.onnx` extension).
 
 You can export your quantized model in the following quantization formats:
 * Fake Quant (where weights and activations are float fakely-quantized values)
 * INT8 (where weights and activations are represented using 8bits integers)
-
-The quantization format value set in the target platform (an edge device with dedicated hardware).
-For more details, please read the [TPC README](../target_platform_capabilities/README.md).  
-```python
-import model_compression_toolkit as mct
-from model_compression_toolkit.target_platform_capabilities.target_platform.quantization_format import \
-    QuantizationFormat
-
-tp = mct.target_platform
-
-def define_tpc(hardware_config, hardware_name):
-    hardware_configuration_options = tp.QuantizationConfigOptions([hardware_config])
-    # Define target platform model
-    hardware_tpc = tp.TargetPlatformModel(hardware_configuration_options, name=hardware_name)
-    
-    # Set the QuantizationFormat suits the hardware: QuantizationFormat.FAKELY_QUANT or QuantizationFormat.INT8
-    with hardware_tpc:
-        hardware_tpc.set_quantization_format(QuantizationFormat.FAKELY_QUANT)
-```
+* MCTQ (where weights and activations are quantized using mct-quantizers custom quantizers).
```

Comment:
where weights and activations are quantized using custom quantizer layers - [mct_quantizers](https://github.com/sony/mct_quantizers#readme)